SimpleChat/Application/Scenes/Home/ViewModels/HomeViewModel.swift

```diff
@@ -25,6 +25,8 @@ final class HomeViewModel {
     init(contactsPermissionRequester: PermissionRequestable, addressBook: AddressBookProvider) {
         self.contactsPermissionRequester = contactsPermissionRequester
         self.addressBook = addressBook
+        
+        isContactsPermissionGranted.value = contactsPermissionRequester.isPermissionGranted
```

Comment:
Initial value allows others to know immediately whether the app can access contacts or not.